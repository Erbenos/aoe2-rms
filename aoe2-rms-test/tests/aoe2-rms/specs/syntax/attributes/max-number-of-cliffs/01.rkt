#lang aoe2-rms
; Create 3-6 cliffs on your map.

<CLIFF-GENERATION>
(max-number-of-cliffs 7)
