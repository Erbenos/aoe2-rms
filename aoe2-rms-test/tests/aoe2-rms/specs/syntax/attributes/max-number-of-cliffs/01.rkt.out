<CLIFF_GENERATION>
max_number_of_cliffs 7
